Services/Blogs/BlogPostService.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Andculture.Sitefinity.Testing.Models.Configuration;
+using Andculture.Sitefinity.Testing.Models.Content.Blogs;
+
+namespace Andculture.Sitefinity.Testing.Services.Blogs
+{
+    public class BlogPostService : ODataService<BlogPost>
+    {
+        public BlogPostService(TestingSettings testingSettings, string accessToken)
+            : base(testingSettings, accessToken)
+        {
+            EndpointUrl = "/sf/system/blogposts";
```

Comment:
Suggestion: Consider moving this path to a configuration store. That way we can quickly update it if it changes over time.